documentation/modules/running-migration-plan.adoc

```diff
@@ -14,19 +14,21 @@ You can run a migration plan and view its progress in the {ocp} web console.
 
 .Procedure
 
-. Click *Migration plans*.
+. In the {ocp} web console, click *Migration* -> *Plans for virtualization*.
 +
-The *Migration plans* list displays the source and target providers, the number of virtual machines (VMs) being migrated, and the status of the plan.
+The *Plans* list displays the source and target providers, the number of virtual machines (VMs) being migrated, the status, and the description of each plan.
 
 . Click *Start* beside a migration plan to start the migration.
+. Click *Start* in the confirmation window that opens.
++
+The *Migration details by VM* screen opens, displaying the migration's progress
 +
 Warm migration only:
 
 * The precopy stage starts.
 * Click *Cutover* to complete the migration.
 
-. Expand a migration plan to view the migration details.
+. If the migration fails, click *Get logs* to view the migration logs, then click *Get logs* in the confirmation window that opens.
```

Comment:
that's incomplete - after clicking "Get logs" twice, the user needs to wait until the button changes from "Get logs" to "Download logs" and then when they press it, they download the logs to their machine